docs/getting-started.md

```diff
@@ -50,6 +50,35 @@ checked-in [deployment yaml](/build/yamls/antrea.yml):
 kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea.yml
 ```
 
+### Deploying Antrea on a Cluster with Existing CNI
+
+Ideally Antrea should only be deployed on a new cluster, which is not already
+running a CNI network plugin. If you need to migrate your existing cluster from
+another CNI to Antrea, you will need to do the following:
+ * Delete previous CNI, including all resources (K8s objects, iptables rules,
+ interfaces, ...) created by that CNI.
+ * Deploy Antrea.
+ * Restart all Pods (you may use `kubectl drain` to drain each Node or restart
```

Comment:
To be more accurate, this is only for Pods in CNI network (not hostNetwork mode).

Reply:
Done